### DISCUSSION TITLE
文件名消失的一种想法，不知道可不可行

### DISCUSSION THREAD

**Discussion by lzy755277** [Jeric-X/SyncClipboard#139] [Ideas]
https://github.com/Jeric-X/SyncClipboard/issues/137，我想问一下能不能通过快捷指令将剪贴板的文件下载到手机中，修改名字，再拷贝那个文件实现。

**Comment by Jeric-X** [comment]
我试过，你也可以自己尝试修改一下





------------------ 原始邮件 ------------------
发件人: righteous ***@***.***&gt;
发送时间: 2024年10月19日 14:43
收件人: Jeric-X/SyncClipboard ***@***.***&gt;
抄送: Subscribed ***@***.***&gt;
主题: Re: [Jeric-X/SyncClipboard] 文件名消失的一种想法，不知道可不可行 (Discussion #139)





 
https://github.com/Jeric-X/SyncClipboard/issues/137，我想问一下能不能通过快捷指令将剪贴板的文件下载到手机中，修改名字，再拷贝那个文件实现。
